### PR TITLE
Remove XML tagging from org.dogtagpki.common

### PR DESCRIPTION
**Pull request by ckelleyRH**

### CODE REVIEW

File: base/tools/src/main/java/com/netscape/cmstools/tps/config/ConfigModifyCLI.java
Comment (edewata) [inline base/tools/src/main/java/com/netscape/cmstools/tps/config/ConfigModifyCLI.java:93]: This is going to change the input format for `pki tps-config-modify --input <file>`:
https://github.com/dogtagpki/pki/wiki/PKI-TPS-Configuration-CLI

@06shalini Is it OK if we change this CLI to take a JSON file instead of XML file?

Comment (edewata) [reply base/tools/src/main/java/com/netscape/cmstools/tps/config/ConfigModifyCLI.java:93]: @ckelleyRH We'll need to move that page into the source repo so we can maintain different docs for different PKI versions. So PKI 10.11 will continue to use XML, but PKI 11.0 will use JSON.

File: base/server/src/main/java/org/dogtagpki/server/rest/InfoService.java
Comment (edewata) [inline base/server/src/main/java/org/dogtagpki/server/rest/InfoService.java:62]: Should be `jsonInfo` :)

File: base/tools/src/main/java/com/netscape/cmstools/tps/config/ConfigModifyCLI.java
Comment (edewata) [reply base/tools/src/main/java/com/netscape/cmstools/tps/config/ConfigModifyCLI.java:93]: I'm moving the doc into the master branch: https://github.com/dogtagpki/pki/pull/3616. It will need to be cherry-picked into `v10.11` branch as well. Once it's merged, we can update this PR (#3610) to update the doc as well.

File: base/server/src/main/java/org/dogtagpki/server/rest/InfoService.java
Comment (ckelleyRH) [reply base/server/src/main/java/org/dogtagpki/server/rest/InfoService.java:62]: Yep :+1:

**Review by edewata — APPROVED** [review]
I created a PR #3643 to update the doc. That PR and this one should be merged together.

**Comment by ckelleyRH** [comment]
@edewata thanks! I'll wait for the approval on the other PR before merging.

**Comment by edewata** [comment]
@ckelleyRH I'll merge both PRs. Thanks!

**Pull request merged**
